hasura.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script fetches a docker-compose.yml, then deploys Hasura GraphQL Engine and Postgres database using Docker Compose.
+
+# Download docker-compose.yml
+if [ -f docker-compose.yml ]
+then 
+  echo -e "\n\033[1;32m==== docker-compose.yml present ====\033[0m\n"
+else
+  echo -e "\n\033[1;33m==== Curling docker-compose.yml ====\033[0m\n"
+  sudo curl https://raw.githubusercontent.com/hasura/graphql-engine/stable/install-manifests/docker-compose/docker-compose.yaml -o docker-compose.yml
+fi
+
+# Start Hasura GraphQL Engine and Postgres database in Docker containers
+if [ -f docker-compose.yml ]
```

Comment:
you have this one backwards, you need to check if hasura is running and if not run it.